build.gradle

```diff
@@ -123,3 +157,22 @@ tasks.register('resetloop') {
     dependsOn restartloop
 }
 
+def pyisort = tasks.register("pyisort", PythonTask) {
+    workDir = "service-python"
+    module = 'isort'
+    command = ". --check --skip tests"
+}
+
+def pyflake8 = tasks.register("pyflake8", PythonTask) {
+    workDir = "service-python"
+    module = 'flake8'
+    command = "."
+    dependsOn pyisort
+}
+
+tasks.register("pytest", PythonTask) {
+    workDir = "service-python"
+    module = 'pytest'
+    command = "--cov-report term-missing --cov=. --cov-fail-under=80 --cov-config=../.coveragerc"
+    dependsOn pyflake8
+}
```

Comment:
Nice! Since all these are running in the `service-python` directory, I'm wondering if these would be in a `build.gradle` file in the `service-python` directory. Then it can be run with `./gradlew service-python:pytest`, or can be triggered to run when `./gradlew test` is run by adding a gradle test dependency `test.dependsOn(pytest)` (or using something like https://github.com/datastax/junitpytest).

Reply:
Added